validation/src/main/scala/hmda/validation/engine/ts/validity/TsValidityEngine.scala

```diff
@@ -2,29 +2,25 @@ package hmda.validation.engine.ts.validity
 
 import hmda.model.fi.ts.TransmittalSheet
 import hmda.validation.api.ValidationApi
+import hmda.validation.engine.ValidationError
 import hmda.validation.engine.ts.TsCommonEngine
+import hmda.validation.rules.EditCheck
 import hmda.validation.rules.ts.validity.{ V105, V140, V155 }
 
-trait TsValidityEngine extends TsCommonEngine with ValidationApi {
-
-  private def v105(t: TransmittalSheet): TsValidation = {
-    convertResult(t, V105(t), "V105")
-  }
+import scalaz._
 
-  private def v140(t: TransmittalSheet): TsValidation = {
-    convertResult(t, V140(t), "V140")
-  }
+trait TsValidityEngine extends TsCommonEngine with ValidationApi {
 
-  private def v155(t: TransmittalSheet): TsValidation = {
-    convertResult(t, V155(t), "V155")
+  private def doCheck[T](editCheck: EditCheck[T], input: T): ValidationNel[ValidationError, T] = {
```

Comment:
Same comment as above, plus an additional observation. This method uses a parameterized type, while the one in `LarValidityEngine` doesn't. Why the difference? Is this method a candidate for abstraction for both? (i.e. in `ValidationApi`)